Targets/Windows/RDPJumplist.tkape

```diff
@@ -0,0 +1,14 @@
+Description: RDP Jumplist Files
```

Comment:
https://github.com/EricZimmerman/KapeFiles/blob/master/Targets%2FWindows%2FLNKFilesAndJumpLists.tkape

Would this be better served adding to the preexisting Target?

Reply:
If you consider it better to add to the preexisting Target then no problem. The extracted folders contain also other files not only jumplist. This is why I ad it as a separate target. :)